src/main/java/edu/ucsb/cs156/organic/controllers/UsersController.java

```diff
@@ -42,6 +40,22 @@ public ResponseEntity<String> users()
         return ResponseEntity.ok().body(body);
     }
 
+    @Operation(summary= "Toggle a user's instructor status")
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @PostMapping("/toggleInstructor")
+    public Object postUsersToggleInstructor(
```

Comment:
would it make more sense to take in the desired value of the instructor field (true, false) instead of just toggling to the opposite?

Reply:
Possibly, but the entire rest of the code base is built around the toggle feature at the moment, so it's better to be consistent.